docs/docs/installation/helm.md

```diff
@@ -0,0 +1,35 @@
+| controllerManager.resources                                     | The resource request/limits for the container image                                                                                                                                                                                                 | limits: 0.1 CPU, 30Mi, requests: 0.1 CPU, 20Mi                            |
+| controllerManager.nodeSelector                                  | The node selector to use for pod scheduling                                                                                                                                                                                                         | `kubernetes.io/os: linux`                                                 |
+| controllerManager.tolerations                                   | The tolerations to use for pod scheduling                                                                                                                                                                                                           | `[]`                                                                      |
+| controllerManager.affinity                                      | The node affinity to use for pod scheduling                                                                                                                                                                                                         | `{}`                                                                      |
+| eraser.image.repository                        | Image repository for worker                                                                                                                                                                                                                                    | `ghcr.io/azure/eraser`   |
+| eraser.image.tag                               | Image tag for worker                                                                                                                                                                                                                                    | Current release version: `v0.1.0` |
+| nameOverride                                  | Override name if needed                                                                                                                                                                                                                | `""`                                                                      |
```

Comment:
Should we link to https://github.com/Azure/eraser/tree/main/charts/eraser otherwise we'll need to autogenerate this